build.gradle

```diff
@@ -19,6 +19,10 @@ dependencies {
     compile 'io.swagger:swagger-annotations:1.5.0'
     compile 'javax.validation:validation-api:2.0.1.Final'
 
+    compile 'javax.xml.bind:jaxb-api:2.3.0'
+    compile group: 'org.springframework', name: 'spring-core', version: '5.2.6.RELEASE'
```

Comment:
is this used anywhere?

Reply:
i saw its used for md5, is there something we can use instead that will give us just the md5 stuff instead of all the spring stuff